packages/frontend-shared/README.md

```diff
@@ -18,26 +22,143 @@ For the best development experience, you will want to use VS Code with the [Vola
+
+### Router
+
+Only `@packages/app` has a router, so details are described in its [README](../app/README.md).
+### Styles
+
+#### Tailwind and Windi
+We use [Tailwind](https://tailwindcss.com/) through [WindiCSS](https://windicss.org/). The codebase is utility-driven and all CSS that can be achieved through utility classes is written that way. The main way to reuse CSS in multiple places is to extract a component that applies the utility classes and can wrap other elements as needed.
+
+#### Explicit Pixel Values
+WindiCSS can create CSS classes as build time based on what class names we use in our components. That means syntax like this will work:
+
+`<p class="p-20px">`
+
+This allows us to specify explicit pixel values for measurements. We follow this pattern throughout the Cypress App codebase.
+
+### Icons
+
+The icons will temporarily live in this package and will soon be moved to `packages/frontend-shared`.
+
+## Custom Icon Library
+Cy has a very custom icon library, to meet the following needs:
+
+* Most of our icons are duo-tone
+* They must be styled in both dark and light contexts (e.g. on a dark menu bar vs on a light background).
+* Since they're duotone, you want to target the specific strokes and fills of the SVGs to color them.
+* All of the niceties of our utility classes should work (e.g. `group-hover` or `group-focus`).
+
+### Usage
+#### Importing Icons
+1. `import MyIcon from '~icons/cy/path-to-icon_x16'`
+2. Automatic template discovery
+```jsx
+/* This just works. No imports necessary */
+<i-cy-path-to-icon_x16 />
+```
+
+#### Styling Icons
+```jsx
+/* This renders a book icon from `./src/assets/book_x16.svg`
+ * and makes it pink and purple
+ */
+
+<i-cy-book_x16 class="
+  icon-dark-pink-300
+  icon-dark-purple-50
+  hover:icon-dark-pink-500
+  hover:icon-light-purple-300
+" />
+```
+
+### Implementation: Custom classes w/ a WindiCSS plugin
+To support selecting specific paths while keeping Tailwind's incredibly helpful interaction helpers (e.g. `group-hover` or `group-focus`), we use a WindiCSS plugin. Windi configuration lives in the [windi.config.ts](windi.config.ts) file in this package.
+#### Adding new icons
+
+To add an new icon: 
+
+1. Export the icon from Figma (all icons should come from the design system in Figma) as an SVG. 
+1. Add the SVG to the [icons folder](./src/assets/icons). 
+1. Name the file following the existing convention in there: `icon-name_x[size in pixels]`, e.g. `arrow-down_x16.svg`.
+1. Manually edit the SVG file to add classes `icon-dark` and `icon-light` to the dark and light internal elements, and save the file. If an icon path doesn't define a class, nothing bad will happen, it just won't get targeted by any styling.
+1. Finally, you don't need to expose anything. `./src/assets/icons` is automatically watched and loaded 😮
+
+Now the icon is ready to be used in Vue SFC templates  
+### Accessibility
```

Comment:
Should this be a separate `## Accessibility` rather than part of `## Custom Icon Library`?